pkgs/development/compilers/scas/default.nix

```
@@ -0,0 +1,22 @@
+{ fetchFromGitHub, stdenv, cmake }:
+
+
+stdenv.mkDerivation rec {
+  pname = "scas";
+  version = "0.4.6";
+  src = fetchFromGitHub {
+    owner = "KnightOS";
+    repo = "scas";
+    rev = "${version}";
+    sha256 = "1c6s9nivbwgv0f8n7j73h54ydgqw5dcpq8l752dfrnqg3kv3nn0h";
+  };
+  nativeBuildInputs = [ cmake ];
+  installTargets = [ "install" ];
+  hardeningDisable = [ "format" ];
+    meta = with stdenv.lib; {
```

```suggestion
  meta = with stdenv.lib; {
```
Tiny nitpick here, meta has the wrong indentation level.